tests/cypress/integration/actions_tasks/case_120_change_completed_jobs_task.js

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2021-2022 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Task change completed jobs.', () => {
+    const caseId = '120';
+    const labelName = `Case ${caseId}`;
+    const taskName = `New annotation task for ${labelName}`;
+    const attrName = `Attr for ${labelName}`;
+    const textDefaultValue = 'Some default value for type Text';
+    const imagesCount = 15;
+    const imageFileName = `image_${labelName.replace(' ', '_').toLowerCase()}`;
+    const width = 800;
+    const height = 800;
+    const posX = 10;
+    const posY = 10;
+    const color = 'gray';
+    const archiveName = `${imageFileName}.zip`;
+    const archivePath = `cypress/fixtures/${archiveName}`;
+    const imagesFolder = `cypress/fixtures/${imageFileName}`;
+    const directoryToArchive = imagesFolder;
+    const advancedConfigurationParams = {
+        multiJobs: true,
+        segmentSize: 5,
+    };
+
+    before(() => {
+        cy.imageGenerator(imagesFolder, imageFileName, width, height, color, posX, posY, labelName, imagesCount);
+        cy.createZipArchive(directoryToArchive, archivePath);
+        cy.visit('auth/login');
+        cy.login();
+        cy.createAnnotationTask(
+            taskName,
+            labelName,
+            attrName,
+            textDefaultValue,
+            archiveName,
+            false,
+            advancedConfigurationParams,
+        );
+    });
+
+    after(() => {
+        cy.goToTaskList();
+        cy.deleteTask(taskName);
```

Comment:
Better to delete task in headless mode, like a kind of routine work. We do not test deleting a task here

Reply:
You can see how it can be done in some other tests, like:
```
cy.getAuthKey().then((response) => {
            const authKey = response.body.key;
            cy.request({
                method: 'DELETE',
                url: `/api/tasks/${taskID}`,
                headers: {
                    Authorization: `Token ${authKey}`,
                },
            });
        });
```

We could also have a command like ``deleteTaskHeadless`` with this code

Reply:
I will change all the tests I created to this method